src/util/time.rs

```diff
@@ -182,6 +184,17 @@ mod inner {
         // TODO Add monotonic coarse clock time impl for macos and windows
         monotonic_raw_now()
     }
+
+    pub fn realtime_now() -> Timespec {
```

Comment:
please add test

Reply:
I suggest still using SystemTime directly, you can add this later if possible.

Reply:
okay.